.github/workflows/pr.yml

```diff
@@ -34,24 +34,19 @@ jobs:
           flutter-version: ${{ env.FLUTTER_VERSION }}
           channel: 'stable'
 
-      - name: Analyze Dart code
-        run: |
-          cd example
-          flutter analyze
-
-      - name: Get dependencies
+      - name: Get Dependencies
```

Comment:
We shouldn't let the CI go forward if there are any errors or warnings, so this should be back